klippy/extras/probe.py

```diff
@@ -214,10 +226,16 @@ def cmd_PROBE_ACCURACY(self, gcmd):
                              speed, lift_speed))
         # Probe bed sample_count times
         self.multi_probe_begin()
+        discards = 0
         positions = []
         while len(positions) < sample_count:
             # Probe position
             pos = self._probe(speed)
+            # Discard the first readings (probably at most one)
+            if discards <= samples_discard_first:
+                gcmd.respond_info("Probe sample discarded")
+                discards += 1
+                continue
```

Comment:
This continue will skip the retract down the loop that must happen to avoid "Probe triggered prior to movement" error

Reply:
Thanks, I believe I've fixed that.